app/js/lib/foxbox/foxbox.js

```diff
@@ -500,6 +500,12 @@ export default class Foxbox extends Service {
       `${this[p.net].origin}/api/v${this[p.settings].apiVersion}/services`)
       .then((services) => {
         return services.map((service) => {
+          // Some service don't have name, but can have product_name instead,
+          // eventually we should get rid of this line and use unified service
+          // label field provided by taxonomy.
+          service.properties.name = service.properties.name ||
```

Comment:
There is a high chance (at the moment I think it's 100% chance) that OpenZWave devices won't have name, but there is a much higher chance that they have product_name. We agreed with @julienw that it will be easier to fix this on client side for the demo.